seed/compliance-memory/FCA-RECORDKEEPING-SANDBOX-v1.md

```
@@ -0,0 +1,136 @@
+| Поиск и восстановление | Graphiti temporal KG + ClickHouse |
+| Backup | [PLACEHOLDER] |
+| Access logs | pgAudit на всех PostgreSQL БД |
+| Encryption at rest | [PLACEHOLDER] |
+
+---
+
+## Audit Trail минимальный состав
+
+Каждая финансовая запись должна содержать:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)